benchtop/src/backend.rs

```diff
@@ -49,4 +49,11 @@ impl DB {
             DB::Nomt(db) => db.execute(timer, workload),
         }
     }
+
+    pub fn print_metrics(&self) {
```

Comment:
for a follow-up: docs on public items.

I would also prefer to have an approach like:

```
impl Nomt {
  fn metrics(&self) -> Metrics { ... }
}

// user code
nomt.metrics().print();

// or
send_to_prometheus(nomt.metrics())

// or whatever
```